lib/lighthouse/veteran_verification/constants.rb

```diff
@@ -2,6 +2,10 @@
 
 module VeteranVerification
   module Constants
+    ERROR_MESSAGE = [
```

Comment:
Is there a reason the message is an array of the string? Is the intention for `response['data']['message']` to be an array instead of a string? Also, is there a json schema for this? I'm curious if the schema says message is an array.

Reply:
Ya, the messages are set as arrays because there are multiple messages that have to be separated in the UI for some of the possible issues. This `ERROR_MESSAGE` is the first and only one that does not have multiple messages that would need to be separated in the UI.